core/station/api/spec.did

```diff
@@ -601,6 +601,10 @@ type SystemUpgradeOperationInput = record {
   module_extra_chunks : opt WasmModuleExtraChunks;
   // The initial argument passed to the new wasm module.
   arg : opt blob;
+  // Determines whether a backup snapshot should be taken (before the upgrade).
+  // If so, the last backup snapshot (if one exists) is atomically replaced
```

Comment:
What does "atomically" mean here?

Reply:
Atomically means that the old snapshot is only deleted if a new one is successfully taken, i.e., you won't end up with no snapshot if taking a new snapshot fails.